server.py

```diff
@@ -169,27 +213,88 @@ async def state():
                 }
     return  { "state": State.INTERLUDE }
 
+@app.post("/play/file")
+async def play_file(file_path: str, title: str, thumbnail: str):
```

Comment:
i think you could combine this and `/cache/play/file`, where if we want to play the entire cache, you can just call `/play/file` with the file_path being set to `"cache"`

Reply:
resolved